### PR TITLE
Add support for Symfony/YAML 4.4

### PR DESCRIPTION
**Pull request by recchia**
Add support for Symfony/YAML 4.4, this is a transition version to 5. In 4.4 you can fix/remove deprecated code before migrate to version 5

### CODE REVIEW

**Comment by aberlinski** [comment]
Hello @cebe 

Could you please tell me when this pull request is going to be merged? We want to use cebe/php-openapi in our project, but lack of symfony/Yaml 4.4 support blocks us. I would be appreciate to merge it as quick as it possible :)

**Comment by cebe** [comment]
I was explicitly excluding this version because of bugs in the yaml implementation, see https://github.com/cebe/php-openapi/issues/60

If it does not break my tests I can add it back, need to check it.

File: composer.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
 	"require": {
 		"php": ">=7.1.0",
 		"ext-json": "*",
-		"symfony/yaml": "^3.0 | ~4.0.0 | ~4.1.0 | ~4.2.0 | ~4.3.0 | ^5.0",
+		"symfony/yaml": "^3.0 | ~4.0.0 | ~4.1.0 | ~4.2.0 | ~4.3.0 | ~4.4.0 | ^5.0",
```
Comment (cebe) [inline composer.json:23]:
```suggestion
		"symfony/yaml": "^3.0 | ^4.0 | ^5.0",
```

**Comment by cebe** [comment]
updated composer.json, lets see if tests are green.

**Comment by cebe** [comment]
https://travis-ci.org/github/cebe/php-openapi/builds/691798677

**Comment by cebe** [comment]
Merged, Thank you!

**Pull request merged**